cms/articles/tests/test_pages.py

```diff
@@ -61,6 +61,19 @@ def test_previous_releases_article_list(self):
         self.assertContains(response, first_article.title)
         self.assertContains(response, second_article.title)
 
+    def test_previous_releases_breadcrumbs(self):
+        """Test that the previous releases page includes a breadcrumb for the latest article."""
+        _latest_article = StatisticalArticlePageFactory(parent=self.page, title="Latest Article")
```

Comment:
NP: We have prior art where we don't assign the result to an unused variable and just rely on the side effect, so we can stick to doing this:

https://github.com/ONSdigital/dis-wagtail/blob/283b88b84cc197440fe861438f3d2e9b3d61c449/cms/articles/tests/test_models.py#L167-L175

Reply:
Good catch, that was a lazy fix to linting complaints, fixed in https://github.com/ONSdigital/dis-wagtail/pull/368/commits/b07eca106e9092a86139f459413db92c82e314ef